### ISSUE TITLE
promise outcome not getting logged. RHEL 7 and Cfengine-3.10-1

### ISSUE THREAD

**Issue by MadhuChittam** [evolvethinking/delta_reporting#40]
promise outcome not logged , only below line in /var/cfengine/delta_reporting/log/promises
BEGIN 2017-06-12T05:37:32-0500
copied evolve_freelib.cf and efl_common.cf to masterfiles/lib/EFL
and efl.cf to masterfiles/services/autorun
followed install document and created efl_data and other required files.

**Comment by neilhwatson** [comment]
See issue #32

**Comment by MadhuChittam** [comment]
yes, tried the solution in issue 32 and also was using efl_main method.
promise is getting executed but NOT logging to promises  file

**Comment by MadhuChittam** [comment]
what efl bundles we need to specify in promises.cf file ?

**Comment by MadhuChittam** [comment]
here are my files. i am just trying to setup delta reporting using examples.

[efl-setup.zip](https://github.com/neilhwatson/delta_reporting/files/1075233/efl-setup.zip)

**Comment by neilhwatson** [comment]
Logs 'classes' and 'promises' are temps and change during the run. Do not look at them. Both the 'final' and the downloaded log '10.7.64.59.log' look fine to me.

**Comment by MadhuChittam** [comment]
test promise outcome is not logged. anything i am missing ?

**Comment by neilhwatson** [comment]
Confirm though outputs log or agent verbose output that your expected promises are running. Then look for evidence in the 'final' log on the agent host.
